src/components/scenes/CryptoExchangeScene.tsx

```diff
@@ -400,9 +415,7 @@ export class CryptoExchangeComponent extends React.Component<Props, State> {
             focusMe={this.focusFromWallet}
             onNext={this.onNext}
           >
-            {this.props.hasMaxSpend && (
-              <MiniButton alignSelf="center" label={s.strings.string_max_cap} marginRem={[1.2, 0, 0]} onPress={this.props.exchangeMax} />
-            )}
+            {this.props.hasMaxSpend && <MiniButton alignSelf="center" label={s.strings.string_max_cap} marginRem={[1.2, 0, 0]} onPress={this.onMax} />}
```

Comment:
Optional: Since you are editing this line, it might be good to fix it to `hasMaxSpend ? <MiniButton ... > : null`. That way we are being explicit about rendering or not rendering, vs abusing the `&&` operator.